src/components/calculator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="display">
+      0
+    </div>
+    <button type="button" className="clear">AC</button>
+    <button type="button" className="plus-minus">+/-</button>
+    <button type="button" className="percent">%</button>
+    <button type="button" className="divide operator">/</button>
```

Comment:
- [ ] Kindly use the division sign as presented in the design: &div;. Consider using HTML entity code, `&div;` if you do not have the sign on your keyboard.